gno.land/pkg/integration/defaultsigner.go

```diff
@@ -0,0 +1,32 @@
+package integration
+
+import (
+	"github.com/gnolang/gno/gno.land/pkg/gnoland"
+	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
+	"github.com/gnolang/gno/tm2/pkg/std"
+)
+
+func SignTxs(txs []gnoland.TxWithMetadata, chainID string) error {
```

Comment:
Comment missing

Reply:
Please don't make this function do magic -- you should give at least the keybase instance for signing the transaction, or even better, a `PrivKey` reference, since we don't use ledger signing in integration tests

Reply:
I passed a PrivKey on 2a67769b4f165ffd4d9ed62cf27891387fa8b949